pkg/server/controller.go

```diff
@@ -256,6 +256,15 @@ func (ctrl *Controller) serverMux() (http.Handler, error) {
 		ctrl.drainMiddleware,
 		ctrl.authMiddleware(nil))
 
+	deleteAppRouter := r.Path("/delete-app").Subrouter()
+	deleteAppRouter.Use(
+		api.AuthMiddleware(nil, ctrl.authService, ctrl.httpUtils),
+		authz.NewAuthorizer(ctrl.log, httputils.NewDefaultHelper(ctrl.log)).RequireOneOf(
+			authz.Role(model.AdminRole),
+			authz.Role(model.AgentRole),
```

Comment:
Agent too? To me it would be only Admin, to keep in the same veins as the socket admin server.